Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM openjdk:8-jdk-alpine as builder
+
+RUN mkdir -p /gradle
+
+ADD  build.gradle /gradle
+ADD  settings.gradle /gradle
+ADD  src /gradle/src
+ADD  gradlew /gradle
+ADD  gradle /gradle/gradle
+WORKDIR /gradle
+RUN ./gradlew build
+
+FROM adoptopenjdk/openjdk8-openj9:x86_64-alpine-jdk8u181-b13_openj9-0.9.0-slim
+
+ENV POSTGRES_HOST localhost
+ENV DATASOURCE_USERNAME postgres
+ENV DATASOURCE_PASSWORD postgres
+ENV SERVER_PORT 8091
+ENV FACEBOOK_CLIENT_ID clientId
+ENV FACEBOOK_CLIENT_SECRET clientSecret
```

Comment:
How `clientSecret` will appear here ? I mean, it's environment variable which will be loaded to that dockerfile through some .env file or how it works ?